webapi/src/main/scala/org/knora/webapi/slice/common/service/PredicateObjectMapper.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright © 2021 - 2023 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.common.service
+
+import zio._
+
+import dsp.errors.InconsistentRepositoryDataException
+import org.knora.webapi.IRI
+import org.knora.webapi.messages.store.triplestoremessages.LiteralV2
+import org.knora.webapi.messages.store.triplestoremessages.SparqlExtendedConstructResponse.ConstructPredicateObjects
+import org.knora.webapi.slice.resourceinfo.domain.IriConverter
+
+/**
+ * The [[PredicateObjectMapper]] is a service which provides methods to extract values from a [[ConstructPredicateObjects]].
+ *
+ * @param iriConverter A service that maps between internal and external IRIs.
+ */
```

Comment:
I think that would have been enough for me to get some context what this service does. The docstrings on the single methods are almost tautological